docs/data/charts/lines/lines.md

```diff
@@ -52,6 +52,16 @@ By default, they are stacked in the order you defined them, with positive values
 
 For more information, see [stacking docs](/x/react-charts/stacking/).
 
+## Partial data
+
+You can handle lines with missing data or data starting at different points by providing `null` values.
+
+By default, the tooltip does not show series if they have no value.
```

Comment:
IMHO, `it` is a bit more correct in this case than `they` as we are referring to a single series, are we not? 🤔 
```suggestion
By default, the tooltip does not show a series if it has no value.
```

Reply:
The fun of "series" being plural and singular :)

I was thinking about plural like "it does not the set of series that have no value" to be coherent with the plural of "You can handle lines with missing data" in the previous paragraph.